.buildkite/scripts/build_docker_image.ps1

```diff
@@ -46,11 +46,19 @@ try {
     $version = "$($pathParts[-2])-$($pathParts[-1])"
     
 @"
+# escape=``
 FROM microsoft/windowsservercore
 MAINTAINER The Habitat Maintainers <humans@habitat.sh>
 ADD rootfs /
 WORKDIR /src
-RUN /hab/pkgs/$ident/bin/hab/hab.exe pkg exec core/windows-service install
+SHELL ["powershell", "-Command", "`$ErrorActionPreference = 'Stop'; `$ProgressPreference = 'SilentlyContinue';"]
```

Comment:
@mwrock I yield to your Windows chops, but can we extract this to a stand-alone `Dockerfile` rather than baking it into a script?

I was noticing this about our Linux studio container creation just this morning... the only way it appears that we have to build such containers is embedded into our Buildkite code. If that logic was externalized, it'd be easier for developers and external contributors to run.

Reply:
it's actually pretty easy to create a windows docker studio using this script. I'm going to create a separate issue for removing the old non-buildkite bintray-publish script and add to the studio readme about using this one.

Reply:
see https://github.com/habitat-sh/habitat/issues/6317